cmake/Modules/FindLFS.cmake

```diff
@@ -0,0 +1,153 @@
+# CMake support for large files
```

Comment:
Maybe, but then the equivalent code in libpcap should be marked as "this was picked up by tcpdump and changes should be made there, too".

Or:

- in tcpdump, say "all changes to this should be propagated to the same file in libpcap";
- in libpcap, say "all changes to this should be propagated to the same file in tcpdump".

(And then we should look at tcpslice; *that* does a bunch of seeking, so it may involve even more work there.)